homeassistant/components/flume/sensor.py

```diff
@@ -152,7 +161,25 @@ def update(self):
             self._available = False
             return
         _LOGGER.debug("Successful update of flume sensor: %s", self._name)
-        self._state = self._flume_device.value
+        self._state = self._flume_device.values["current_interval"]
+        self._attributes = {
+            "month_to_date": format_attribute_value(
+                self._flume_device.values["month_to_date"]
+            ),
+            "week_to_date": format_attribute_value(
+                self._flume_device.values["week_to_date"]
+            ),
+            "today": format_attribute_value(self._flume_device.values["today"]),
+            "last_60_min": format_attribute_value(
+                self._flume_device.values["last_60_min"]
+            ),
+            "last_24_hrs": format_attribute_value(
+                self._flume_device.values["last_24_hrs"]
+            ),
+            "last_30_days": format_attribute_value(
+                self._flume_device.values["last_30_days"]
```

Comment:
Are the attributes measuring liters of water consumption per different intervals? Why should they not be individual sensors?

Reply:
Thank you for the feedback. That is correct, they are different consumption measurements. I have broken it into separate sensors in the latest Push.